dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/fat/src/io/openliberty/microprofile/telemetry/internal_fat/JaxRsIntegrationWithConcurrency.java

```diff
@@ -79,6 +83,9 @@ public class JaxRsIntegrationWithConcurrency extends FATServletClient {
     @Server(SERVER_NAME)
     public static LibertyServer server;
 
+    @ClassRule
+    public static RepeatTests r = MicroProfileActions.repeat(SERVER_NAME, MicroProfileActions.MP60);
```

Comment:
Not repeating this with MP61? A comment might have been a good idea.

Reply:
More to the point, this looks like an important test which should be repeated for MP61.

Reply:
I agree that the TCK covers the test cases in `JaxRsServerAsyncTestServlet`.

However, this class does also run a lot of other tests with the concurrent feature enabled. We have seen some differences in behaviour in other features with concurrent enabled vs. disabled which I think is why we did this.